package.json

```diff
@@ -7,8 +7,10 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon index.js",
     "build": "webpack -d --watch",
+    "dev": "webpack -d --watch && nodemon index.js",
     "seed": "node server/utils/seed.js",
-    "dropDB": "node server/utils/dropDB.js"
+    "dropDB": "node server/utils/dropDB.js",
+    "lintingFE": "./node_modules/.bin/eslint './client/src/**'"
```

Comment:
If the `eslint` isn't working, let's remove it from the package.json because this implementation seems very over complicated.